source/cloud/cloud-administration/sso-google.md

```diff
@@ -2,49 +2,77 @@
 
-4. Select **Credentials** and click **Create credentials**, then choose **OAuth client ID** from the drop-down list.
+3. Scroll further down to add your Mattermost instance to a list of authorized domains that can access the API. For example, if it is hosted on `mattermost.yourdomain.com` add `yourdomain.com`.
 
-![](../../../source/images/google_3_oauth_client_id.png)
+4. Select **Credentials** and select **Create credentials**, then choose **OAuth client ID** from the drop-down list.
 
 5. Select **Web Application** as the **Application type**, and choose a descriptive **Name** for the OAuth connection.
 
-Under **Restrictions > Authorized redirect URIs**, enter `your-mattermost-url/signup/google/complete` (example: `http://localhost:8065/signup/google/complete`). Click **Create**.
+6. Under **Restrictions > Authorized redirect URIs**, enter `your-mattermost-url/signup/google/complete` (example: `http://localhost:8065/signup/google/complete`). Select **Create**.
+
+![](../../../source/images/google_3_oauth_consent_screen.png)
+
+![](../../../source/images/google_authorised_domains.png)
+
+![](../../../source/images/google_3_oauth_client_id.png)
 
 ![](../../../source/images/google_4_web_app.png)
 
-6. Copy the **Client ID** and **Client Secret**, which will be used to configure Google SSO for Mattermost.
+### Step 4: Copying values for Mattermost configuration
+
+Copy and paste the **Client ID** and **Client Secret** values to a temporary location. You will enter these values as a **Client ID** and **Client Secret** in the Mattermost System Console.
 
 ![](../../../source/images/google_5_client_id_secret.PNG)
 
-7. In **System Console > OAuth 2.0 > Select OAuth 2.0 service provider**, choose **Google Apps** as the service provider and enter **Client ID** and **Client Secret** from step 6 in their respective fields.
+### Step 5: Configuring Mattermost for Google Apps SSO
+
+1. Log in to Mattermost, then go to the **System Console > Authentication > OpenID Connect**.
+
+2. Select **Google Apps** as the service provider.
+
+3. Paste in the **Client ID**.
+
+3. Paste in the **Client Secret**.
 
-8. Restart your Mattermost server to see the changes take effect.
+4. Restart your Mattermost server to see the changes take effect.
 
 **Note:**
-- You may also enter **Client ID** and **Client Secret** fields from step 6 directly in `GoogleSettings` section of `config/config.json`.
-- The following default values in `GoogleSettings` section of `config/config.json` are recommended:
+- Alternatively, you may also enter **Client ID** and **Client Secret** values directly into the `GoogleSettings` section of the Mattermost `config/config.json` file.
+- The following default values are recommended:
     - `Scope`: `profile email`
     - `AuthEndpoint`: `https://accounts.google.com/o/oauth2/v2/auth`
     - `TokenEndpoint`: `https://www.googleapis.com/oauth2/v4/token`
```

Comment:
@sbishel - Anything here that needs to change with regards to supporting OpenID Connect over OAuth 2?

Reply:
see above.